README.md

```diff
@@ -27,52 +27,53 @@ delivered in each release and our future plans.
 
 ## Egeria governance
 
-This project aims to operate in a transparent, accessible way for the benefit
-of the Egeria community.
-All participation in this project is open and not
-bound to any corporate affiliation.
+This project aims to operate in a transparent, accessible way for the benefit of the Egeria community.
+All participation in this project is open and not bound to any corporate affiliation.
 
-To understand how to join and contribute see the 
+To understand how to join and contribute, see the 
 [Community Guide](./Community-Guide.md).
 
-All participants are bound the ODPi's [Code of Conduct](https://github.com/odpi/specs/wiki/ODPi-Code-of-Conduct).
+All participants are bound to the ODPi's [Code of Conduct](https://github.com/odpi/specs/wiki/ODPi-Code-of-Conduct).
```

Comment:
Oops. I guess we haven't updated to reference linux foundation yet?

Reply:
I am not sure what we should point to., Will raise an issue.